### DISCUSSION TITLE
Integrate Models to the tool

### DISCUSSION THREAD

**Discussion by keshavadk** [cvat-ai/cvat#4213] [Q&A]
After cloning the CVAT url to develop locally, we are not able to find the "Models" tab as compared to original cvat online tool.
Can i  know, how can we integrate the "Models"?

I have attached the images of local machine and Cvat.org
 
![1_local](https://user-images.githubusercontent.com/16021947/150330269-03d528b0-3ce0-466a-abdc-4fb2fe744541.JPG)
![2_cvat](https://user-images.githubusercontent.com/16021947/150330288-c4ab288e-b0ba-4f66-b9f0-248d33a9eb28.JPG)

Thank you.

**Comment by nmanovic** [comment]
https://openvinotoolkit.github.io/cvat/docs/administration/advanced/installation_automatic_annotation/

**Reply by keshva23** [reply]
Thank you @nmanovic for the reply.
